tools/layout_analysis/sampling_images_for_layout_analysis.py

```diff
@@ -1,24 +1,36 @@
 import os
+import csv
 import boto3
+import shutil
 import hashlib
-from pathlib import Path
 from git import Repo
+from pathlib import Path
 
 
 # s3 config
```

Comment:
i would use the config.py. if config is different for test, i would write a seperate config in test. in future if u want to use similar config in multiple test than current approach would be chaotic

Reply:
duly noted, I will update the config for future PR